upgrade_test.py

```diff
@@ -615,6 +619,8 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
             self.db_cluster.node_to_upgrade.check_node_health()
 
             # wait for the 10m read workload to finish
+            read_sb_large_partitions = stress_cmd_large_partitions_10m.replace('-mode=write', '-mode=read')
+            self.run_stress_thread(stress_cmd=read_sb_large_partitions)
```

Comment:
This should be moved to Step6, "verify_after_upgrade".
In the yaml file, you can use the parameter called: stress_after_cluster_upgrade.

Reply:
fixing